### PR TITLE
Update Traditional Chinese (zh_tw.json)

### PR DESCRIPTION
**Pull request by yichifauzi**
Update Traditional Chinese translation

### CODE REVIEW

File: src/main/resources/assets/bettergrass/lang/zh_tw.json [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
   "bettergrass.moreBlocks.desc": "應該在斜坡上連接的類似草地的方塊。",
-  "bettergrass.betterSnow": "Better Snow",
-  "bettergrass.betterSnowMode": "Better Snow Mode",
-  "bettergrass.betterSnowMode.desc": "§4Off §r- No snow layer will be generated.\n§3OptiFine-like §r- Shows a snow layer beneath specific blocks that have at-least one snow layer or snow block near them.\n§dLambdaBetterGrass-like §r- Shows a snow layer beneath specific blocks that have at-least \"two\" snow layers or snow blocks near them.",
-  "bettergrass.betterSnowMode.optifine": "OptiFine-like",
-  "bettergrass.betterSnowMode.lambda": "LambdaBetterGrass-like",
-  "bettergrass.excludedTags": "Excluded Tags",
-  "bettergrass.excludedBlocks": "Excluded Blocks",
+  "bettergrass.betterSnow": "更好的雪地",
+  "bettergrass.betterSnowMode": "更好的雪地",
```
Comment (UltimatChamp) [inline src/main/resources/assets/bettergrass/lang/zh_tw.json:15]: Can you change this from `Better Snow` --> `Better Snow Mode`?

**Review by UltimatChamp — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/bettergrass/lang/zh_tw.json (hunk 1 above)
Comment (yichifauzi) [reply src/main/resources/assets/bettergrass/lang/zh_tw.json:15]: You mean translation or the string key?

Comment (UltimatChamp) [reply src/main/resources/assets/bettergrass/lang/zh_tw.json:15]: translation

Comment (notlin4) [inline src/main/resources/assets/bettergrass/lang/zh_tw.json:15]:
```suggestion
  "bettergrass.betterSnowMode": "更好的雪地模式",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Review by UltimatChamp — APPROVED** [review]

**Pull request merged**